### DISCUSSION TITLE
how do I configure sketchybar to showcase an `aerospace` workspace?

### DISCUSSION THREAD

**Discussion by baggiponte** [FelixKratz/SketchyBar#598] [Ideas]
[Aerospace](https://github.com/nikitabobko/AeroSpace) is a new tiling window manager for macOS. It uses a new concept of "workspaces", which aren't "spaces"/Desktops. Hence if I am using sketchybar to see what space I am on, while moving windows across aerospace workspaces, nothing will be updated. how can I have this show up in sketchybar? the aerospace command to get the workspace currently in focus is `aerospace list-workspaces --focused`.

**Comment by baggiponte** [comment]
I guess I should RTFM myself: https://nikitabobko.github.io/AeroSpace/goodness#show-aerospace-workspaces-in-sketchybar 🔥

**Reply by Mahkxai** [reply]
@baggiponte, @FelixKratz Nice, thank you! I notice that there's some lag before the sketchyBar updates - is this normal?

**Comment by FelixKratz** [comment]
Also see: https://github.com/FelixKratz/SketchyBar/discussions/599 for a lua example
